### PR TITLE
HDDS-7381. Cleanup of VolumeManagerImpl

### PR DESCRIPTION
**Pull request by myskov**
## What changes were proposed in this pull request?

I removed unused constructor arguments and cleanup javadoc comments (mostly removed duplicated ones)

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-7381

## How was this patch tested?

unit tests

### CODE REVIEW

**Comment by kerneltime** [comment]
@DaveTeng0 @Galsza @dombizita

**Review by DaveTeng0 — COMMENTED** [review]
LGTM!

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/VolumeManagerImpl.java [hunk 1]
```diff
@@ -36,31 +35,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/**
- * OM volume management code.
```
Comment (Galsza) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/VolumeManagerImpl.java:40]: Checkstyle error here, please fix it.

**Review by Galsza — CHANGES_REQUESTED** [review]
One minor checkstyle issue, otherwise LGTM and thanks for the patch!

**Comment by Galsza** [comment]
LGTM

**Comment by dombizita** [comment]
thanks for the patch @myskov, looks good to me!

**Review by adoroszlai — APPROVED** [review]

**Pull request merged**